app/configs/locationColumns.ts

```diff
@@ -0,0 +1,16 @@
+import { ColDataCardDef } from "@/types/types";
+import { Location } from "@/types/locations";
+
+export const locationColumns: {
+  [name: string]: ColDataCardDef<Location>;
+} = {
+  location_id: {
+    name: "location_id",
+    label: "Location ID",
+  },
+  // todo: need to support non-lookup table relationships
+  //   cr3_crash_count: {
+  //     name: "cr3_crash_count",
+  //     relationshipName: "locations_list_view"
+  //   }
+};
```

Comment:
the current data cards will render lookup table relationships, but there is mechanism to, render card values through other relationships, like the `locations` <> `locations_list_view` one-to-one relationship. it felt like a big enough lift to leave separate. and probably not worth touching until we settle the question of if we want to stick with zod-generated types 🤔